build_tools/ci/cpu_comparison/input_generator.py

```diff
@@ -1,33 +1,3 @@
-# This script is expected to be run from the command-line with 3 arguments:
```

Comment:
This is no longer used a a script, this comment predates making the test driver a python script (which now just calls functions directly)